docs/entity-manager-api.md

```diff
@@ -147,6 +147,29 @@ await manager.update(User, 1, { firstName: "Rizzrak" });
 // executes UPDATE user SET firstName = Rizzrak WHERE id = 1
 ```
 
+* `upsert` - Inserts a new entity or array of entities unless they already exist in which case they are updated instead.
```

Comment:
I think we need to left a note what databases support this method

Reply:
@pleerock I've added that